iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/AbstractTransformerSerDe.java

```diff
@@ -151,10 +152,13 @@ protected static void writePhaseTapChanger(String name, PhaseTapChanger ptc, Net
         context.getWriter().writeStartNode(context.getVersion().getNamespaceURI(context.isValid()), name);
 
         RegulationMode regMode = ptc.getRegulationMode();
-        Boolean optionalRegulatingValue = (regMode == null || regMode == RegulationMode.FIXED_TAP) && !ptc.isRegulating() ? null : ptc.isRegulating();
+        Boolean optionalRegulatingValue = !ptc.hasLoadTapChangingCapabilities() || (regMode == null || regMode == RegulationMode.FIXED_TAP) && !ptc.isRegulating() ? null : ptc.isRegulating();
```

Comment:
Shouldn't it be `&&` here, to be consistent with what is done for the ratio tap changers?

For ratio tap changers, it is:
```java
Boolean optionalRegulatingValue = rtc.hasLoadTapChangingCapabilities() || rtc.isRegulating() ? rtc.isRegulating() : null;
```
Therefore, `optionalRegulatingValue` is set to `rtc.isRegulating()` when the tap changer has load tap changing capabilities.

Reply:
Hi @olperr1 @rcourtier 

following the change that regulating implies that ltcFlag is true, proposing to change as follows:

```java
        // RTC
        Boolean optionalRegulatingValue = !rtc.hasLoadTapChangingCapabilities() ? null : rtc.isRegulating();

        // PTC
        Boolean optionalRegulatingValue = !ptc.hasLoadTapChangingCapabilities() || regMode == null || regMode == RegulationMode.FIXED_TAP ? null : ptc.isRegulating();
```

for RTC: don't write regulating if not hasLoadTapChangingCapabilities()
for PTC: same, but with additional consideration of `null` or `FIXED_TAP` modes which would also cause regulating not being written

Reply:
Hi @jeandemanged:
I think you are right, it is clearer with your suggestion.